### DISCUSSION TITLE
Not row-cols-xs-auto works with Cards

### DISCUSSION THREAD

**Discussion by djom202** [twbs/bootstrap#38636] [Q&A]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

I've a set of cards in 4x2 way in my web design but in mobile I'm needing to put them in 8x1 way, so I'm using this class in order to set a list of cards and it seems works with the first 2 card, nevertheless after that I got card in pairs.

My Html:
```html
<div class="row row-cols-xs-auto row-cols-md-4">
    <div v-for="item in props.items" :key="item.title" class="col mb-3 animated">
        <div class="card cristalEffect">
            <div class="card-header customHeader">{{ item.title }}</div>
            <div class="card-body">
                <small class="card-text">{{ item.text }}</small>
            </div>
        </div>
    </div>
</div>
```

On Desktop view:
<img width="1235" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/e0b1747f-0866-4de0-8342-7a9793cf9c09">

On Mobile view:
<img width="311" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/c9378913-468b-4d1f-8902-358ecd784d34">
<img width="316" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/aef80306-b01e-423b-9176-a3d733acc4c8">
<img width="313" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/21ad5eea-8df2-4153-8c08-3a4298f75bd9">

### Reduced test cases

I just install the dependencie from npm and applied the style for the cards

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.2.3

**Comment by julien-deramond** [comment]
Hey @djom202 
I've converted the issue into a Q&A discussion because it seems to be for now a usage issue.
If possible could you please try to create a CodePen, StackBlitz, or other so that we have something already to test in order to help you?

My first feedback is that `.row-cols-xs-auto` doesn't exist by default in Bootstrap. Have you added it by using Sass utilities?

**Comment by djom202** [comment]
Hi @julien-deramond, maybe it was aa confused I saw this code into the documentation:

<img width="547" alt="image" src="https://github.com/twbs/bootstrap/assets/1834808/9b61475d-9170-4fcd-81ce-932417452ce8">

So I supposed to that I could use with auto, nevertheless I checked again the documentation since you said that class not exist, so I fixed the class and it's works.

Thanks a lot.
